beacon-chain/core/epbs/attestation.go

```diff
@@ -0,0 +1,13 @@
+package epbs
+
+import (
+	"errors"
+)
+
+// RemoveValidatorFlag removes validator flag from existing one.
+func RemoveValidatorFlag(flag, flagPosition uint8) (uint8, error) {
+	if flagPosition > 7 {
+		return flag, errors.New("flag position exceeds length")
```

Comment:
what do you think about returning the length as well?
```suggestion
		return flag, fmt.Errorf("flag position %d exceeds length", flagPosition)
```

Reply:
Sounds good! Do you want to modify `HasValidatorFlag` and `AddValidatorFlag` in `core/altair/attestation.go` as well?

Reply:
I dont think we should touch them

Reply:
Followed your suggestion! It's now squashed and rebased.